raw_data/preprocessed/africa/lesotho_english.txt

```diff
@@ -0,0 +1,379 @@
alteration of the physical, thermal, chemical, biological or radio-active properties of a segment of the environment by discharging, emitting or depositing substances or wastes so as to affect any beneficial use adversely, to cause a condition which is hazardous or potentially hazardous to public health, safety, welfare or to animals, birds, wildlife, fish or aquatic life or to plants or cause a contravention of any condition, limitation or restriction which is subject to a licence issued under this Act; “precautionary principle” means that where there are threats of damage to the environment, whether serious or irreversible, lack of full scientific certainty shall not be used as a reason for postponing cost-effective measures to prevent environmental degradation; “Prime Minister” means the Prime Minister of Lesotho; “Principal Secretary” means the Principal Secretary of the Ministry responsible for environment; “project” includes both project and policy that leads to projects which have or are likely to have an impact on the environment; “spill' means a discharge of a pollutant into the environment from or out of a structure, vehicle, vessel, aircraft or other carrier or container, which: [a] is abnormal having regard to all the circumstances of the discharge; and [b] poses a serious threat to the environment and human health; “soil” means an upper layer of earth and includes sand, rock, shales, minerals, vegetation and soil flora, and fauna in the soil and derivatives such as dust; “standard” means the limits of pollution established under this Act or under the regulations made under this Act or any other law; “strategic environmental assessment” means an assessment of the positive and adverse effects that the implementation of Bill, regulation or of a public policy, programme or plan is likely to have on the environment; “sustainable development” means development that meets the needs of the present generation without compromising the ability of future generations to meet their needs by maintaining the carrying capacity of the supporting ecosystems; “sustainable use” means present use of the environment or natural resources which does not compromise or impose on the ability to use the same by future generations or degrade the carrying capacity of supporting ecosystems; “Tribunal” means the Environmental Tribunal established under section 98; “waste” means any substance that may be prescribed as waste or any matter, whether liquid, solid, gaseous, or radio-active, which is discharged, emitted or deposited in the environment in such volume, composition or manner as to cause an alteration of the environment; “water” means drinking water, water used for domestic and industrial purposes and for animals, plants and includes river, stream, watercourse, reservoir, well, dam, canal, channel, lake, swamp, open drain or underground water; and “wetland” means an area permanently or seasonally flooded by water where plants and animals have become adapted. 
```

Comment:
`Act";` still needs to be replaced with `Act;`. `Act;"` still needs to be replaced with `Act;"`.